hudi-common/src/main/java/org/apache/hudi/common/conflict/detection/TimelineServerBasedDetectionStrategy.java

```diff
@@ -60,4 +60,6 @@ public abstract void startAsyncDetection(Long initialDelayMs, Long periodMs, Str
                                            String basePath, Long maxAllowableHeartbeatIntervalInMs,
                                            HoodieStorage storage, Object markerHandler,
                                            Set<HoodieInstant> completedCommits);
+
+  public abstract void stop();
```

Comment:
Maybe we implement `Closable` instead?

Reply:
I tried `Closable` which needs to throw exception and it needs to fix all parent class's close to throw exception